### DISCUSSION TITLE
Immich 1.118.2 twice photo

### DISCUSSION THREAD

**Discussion by leon-ITA** [immich-app/immich#13535] [Q&A]
### The bug

With new version 1.118.2 a NEW single photo in my Nas is loaded twice in Immich, than o see in the app two time the same photo with same name!
It doesn't happen with previous version!

### The OS that Immich Server is running on

Ubuntu 22.04

### Version of Immich Server

1.118.2

### Version of Immich Mobile App

1.118.2

### Platform with the issue

- [X] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Na
```

### Your .env content

```Shell
Na
```

### Reproduction steps

1. Shot a new photo
2. load it into the shared folder on the nas
3. run scan on immich
4. Only the last photo after version upgrade is loaded two times on immich, the previous photo are loaded correctly (one time)
...

**Comment by bo0tzz** [comment]
Please don't just skip over your compose file etc, they're relevant. Are you using external libraries? If so please post your configuration for those as well.
